src/components/BoardHead/BoardHead.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {format} from 'react-string-format'
+
+
+
+function BoardHead(props){
+
+    let minutes = Math.floor(props.time/ 60);
+    let seconds = Math.floor(props.time - minutes * 60);
+    let secondstoformat = seconds < 10 ? format('0{0}',seconds) : seconds
```

Comment:
`secondsToFormat`

Reply:
but nice usage of inline `if` statement

Reply:
use formatting with backticks: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
also - get rid of the magic numbers, and export this logic to util function:
i.e.
```javascript
cosnt NUMBER_OF_SECONDS_DIGITS = 2;
const formatSeconds = (seconds) => {
    return String(seconds).padStart(NUMBER_OF_SECONDS_DIGITS, '0');
}
```